### PR TITLE
Add Cylinder JWT Authentication doc

### PR DESCRIPTION
**Pull request by ltseeley**
Adds the "Cylinder JWT Authentication" feature document to the features
section of the Splinter website.

Signed-off-by: Logan Seeley <seeley@bitwise.io>

### CODE REVIEW

File: community/planning/cylinder_jwt_authentication.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+The Cylinder library provides all the tools necessary to create and verify
+Cylinder JWTs.
+
+### Creating a Cylinder JWT
+[creating-a-cylinder-jwt]: #creating-a-cylinder-jwt
+
+Splinter clients will create JWTs with the following steps:
+
+1. Load a Cylinder `PrivateKey` from the filesystem or another source
+1. Initialize a signing `Context` (such as the Secp256k1 context) that matches
```
Comment (shannynalayna) [inline community/planning/cylinder_jwt_authentication.md:47]: Looks like the numbering is off here

File: community/planning/cylinder_jwt_authentication.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+`Authorization` HTTP header in the format: `Authorization: Bearer
+Cylinder:<jwt>`.
+
+### Verifying a Cylinder JWT
+[verifying-a-cylinder-jwt]: #verifying-a-cylinder-jwt
+
+The Splinter REST API will verify the Cylinder JWT and extract the signer's
+public key through the following process:
+
+1. Parse the Cylinder JWT from the `Authorization` header when the `Bearer
```
Comment (shannynalayna) [inline community/planning/cylinder_jwt_authentication.md:64]: Same numbering issue here

**Review by shannynalayna — CHANGES_REQUESTED** [review]

File: community/planning/cylinder_jwt_authentication.md (hunk 1 above)
Comment (ltseeley) [reply community/planning/cylinder_jwt_authentication.md:47]: Markdown handles the numbering automatically; using `1.` for every entry would allows us to change the order or insert items without having to mess with the numbering.

File: community/planning/cylinder_jwt_authentication.md (hunk 2 above)
Comment (ltseeley) [reply community/planning/cylinder_jwt_authentication.md:64]: See above comment

**Review by shannynalayna — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]

**Pull request merged**